docs/FAQ.md

```diff
@@ -20,17 +20,21 @@ Some known potential causes are:
 - You're using 32-bit python instead of 64-bit. See the answer
   [here](https://stackoverflow.com/a/1405971/224264) for how to tell which you
   have installed.
+- You're using python 3.9. TensorFlow plans to release packages for this as soon
```

Comment:
Revive the 3.8 wording from https://github.com/Unity-Technologies/ml-agents/commit/6eca1f2e828b45bf25c20686191a726c1a6456fc#diff-0bb42a85d04c8b7c9b659307979fd43e8f6e084283a300bfff959adfcbddfd17